afs/templates/views/components/workflows/project-report-workflow/report-template-select.htm

```diff
@@ -62,11 +62,9 @@
                 data-bind="css: { 
                         'active': $parent.templateValue() === id 
                     },
-                    click: function () {
```

Comment:
Why did this get changed?  This was more correct the first time (wrapping it in anonymous function - which ensures the function is not called on initialization; you assign a function vs calling that function) - if you don't do this, you'll get a default value set for the template - which may have been your intention, but if you do it like this, template name doesn't get set - and you get [Object object] in the "summary/download" step.  Try this with a single template in your db and you'll see what I mean.

Reply:
This was an artifact of me trying to debug passing values between workflow steps. I think everything should be sorted now.